sympy/printing/torch.py

```diff
@@ -0,0 +1,190 @@
+from sympy.printing.pycode import AbstractPythonCodePrinter
+from sympy.external import import_module
+import sympy
+
+torch = import_module('torch')
+
+
+class TorchPrinter(AbstractPythonCodePrinter):
+
+    def _get_loop_opening_ending(self, indices):
+        pass
+
+    def _rate_index_position(self, p):
+        pass
+
```

Comment:
Are these used?

Reply:
no -- just belong to superclass. Removed